src/components/TaxPayer/SpouseAndDependent.tsx

```diff
@@ -60,6 +75,27 @@ export const AddDependentForm = (): ReactElement => {
           required={true}
           error={errors.relationship}
         />
+        <LabeledInput
+          register={register}
+          label="Birth year"
+          patternConfig={Patterns.year(control)}
+          name="birthYear"
+          required={false}
+        />
+        <LabeledInput
+          register={register}
+          label="How many months did this person live with you this year?"
+          patternConfig={Patterns.numMonths(control)}
+          name="numberOfMonths"
+          required={false}
+        />
+        <LabeledCheckBox
+          label="Is this person a student"
```

Comment:
Should we use the language "full-time student" here? Some education credits apply to both part-time or full-time students, but this one is just for full-time. If people are uncertain, a full-time student is enrolled in a full course load for at least 5 months of the year (so one full semester is enough).

Reply:
Yes it  seems we should. 

Definition from instructions:

> Student. A student is a child who during any part of 5 calendar
months of 2020 was enrolled as a full-time student at a school
or took a full-time, on-farm training course given by a school or
a state, county, or local government agency. A school includes a
technical, trade, or mechanical school. It doesn't include an
on-the-job training course, correspondence school, or school offering courses only through the Internet.

I will change it to 'full-time' student for now. It seems safe to present this definition to the user, but I don't want the UI to get covered up with modal popups and info bubbles, not sure how to proceed regarding these sorts of terms.